apis/v1alpha1/gateway_types.go

```diff
@@ -482,7 +476,7 @@ type RouteBindingSelector struct {
 	//
 	// Support: Core
 	//
-	// +required
+	// +kubebuilder:validation:MaxLength=253
```

Comment:
Should `Resource` have a MinLength=1 since it's required and not being defaulted?

Reply:
Updated, thanks!